### PR TITLE
chore(starters): Add gatsby-wordpress-libre-starter to starters

### PR DESCRIPTION
**Pull request by shyamlohar**
## Description

Submitting a new Gatsby starter. It's a blog website integrated with WordPress headless cms.

On-line demo is available at:
https://gatsby-wordpress-libre.netlify.com/

chore(starters): add gatsby-attila-theme-starter

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1,3 +1,22 @@
+- url: https://gatsby-wordpress-libre.netlify.com/
+  repo: https://github.com/armada-inc/gatsby-attila-theme-starter
```
Comment (gillkyle) [inline docs/starters.yml]: It looks like this repo points to a different repository than one that would presumably be named `gatsby-wordpress-libre-starter`?

Is this related to #22175?

File: docs/starters.yml [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+  description: A Gatsby theme plugin for creating blogs from headless WordPress CMS.
```
Comment (gillkyle) [inline docs/starters.yml]:
```suggestion
  description: A Gatsby starter for creating blogs from headless WordPress CMS.
```

If this is to be a starter, I think it would make sense to clarify this wording 🙂

File: docs/starters.yml (hunk 1 above)
Comment (shyamlohar) [reply docs/starters.yml]: I have made changes. it was an error from my side 👍 and it is not related to #22175

File: docs/starters.yml (hunk 2 above)
Comment (shyamlohar) [reply docs/starters.yml]: Yeah it makes sense 🙂 I have made changes that you suggested. will do same for #22175
